[16:47:39] [DEBUG] Running client sanity checks...
[16:47:39] [DEBUG] Loaded Extension: interactions.ext.jurigged
[16:47:40] [DEBUG] No setup function found in play
[16:47:40] [DEBUG] Found extension class Music in play: Attempting to load
[16:47:40] [DEBUG] Added callback: Music._menu
[16:47:40] [DEBUG] Added callback: Music._pause
[16:47:40] [DEBUG] Added callback: Music._resume
[16:47:40] [DEBUG] Added callback: Music._skip
[16:47:40] [DEBUG] Added callback: Music._stop
[16:47:40] [DEBUG] Added callback: Music.play
[16:47:40] [DEBUG] Loaded Extension: play
[16:47:41] [DEBUG] No setup function found in askbard
[16:47:41] [DEBUG] Found extension class Bard in askbard: Attempting to load
[16:47:41] [DEBUG] Added callback: Bard._askbard
[16:47:41] [DEBUG] Added callback: Bard._endbard
[16:47:41] [DEBUG] Added callback: Bard.set_bard_channel
[16:47:41] [DEBUG] Loaded Extension: askbard
[16:47:41] [DEBUG] No setup function found in noi_chu
[16:47:41] [DEBUG] Found extension class NoiChu in noi_chu: Attempting to load
[16:47:41] [DEBUG] Added callback: NoiChu._reset
[16:47:41] [DEBUG] Added callback: NoiChu._world_setup
[16:47:41] [DEBUG] Loaded Extension: noi_chu
[16:47:41] [DEBUG] No setup function found in db_refesh
[16:47:41] [DEBUG] Found extension class Database in db_refesh: Attempting to load
[16:47:41] [DEBUG] Loaded Extension: db_refesh
[16:47:41] [DEBUG] No setup function found in help
[16:47:41] [DEBUG] Found extension class Help in help: Attempting to load
[16:47:41] [DEBUG] Added callback: Help._about
[16:47:41] [DEBUG] Added callback: Help.help
[16:47:41] [DEBUG] Loaded Extension: help
[16:47:41] [DEBUG] No setup function found in channel_listen
[16:47:41] [DEBUG] Found extension class ChannelListen in channel_listen: Attempting to load
[16:47:41] [DEBUG] Added callback: ChannelListen._setup
[16:47:41] [DEBUG] Loaded Extension: channel_listen
[16:47:41] [DEBUG] Added callback: on_start
[16:47:41] [DEBUG] 1 callbacks have been loaded from __main__.
[16:47:41] [DEBUG] Added callback: _disconnect
[16:47:41] [DEBUG] Added callback: _on_websocket_ready
[16:47:41] [DEBUG] Added callback: on_autocomplete_completion
[16:47:41] [DEBUG] Added callback: on_autocomplete_error
[16:47:41] [DEBUG] Added callback: on_command_completion
[16:47:41] [DEBUG] Added callback: on_command_error
[16:47:41] [DEBUG] Added callback: on_component_completion
[16:47:41] [DEBUG] Added callback: on_component_error
[16:47:41] [DEBUG] Added callback: on_error
[16:47:41] [DEBUG] Added callback: on_modal_completion
[16:47:41] [DEBUG] Added callback: on_modal_error
[16:47:41] [DEBUG] Added callback: on_resume
[16:47:41] [DEBUG] 12 callbacks have been loaded from Client.
[16:47:41] [DEBUG] Attempting to login
[16:47:41] [DEBUG] Caching ingested rate limit data for: 78bb8553d9352a5a2f89f9def401287a
[16:47:41] [DEBUG] GET /users/@me Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[16:47:41] [DEBUG] GET /users/@me Received 200 :: [0/1000 calls remaining]
[16:47:42] [DEBUG] Caching ingested rate limit data for: d28a133af187e91d71d96a223467ce49
[16:47:42] [DEBUG] GET /oauth2/applications/@me Received 200 :: [999/1000 calls remaining]
[16:47:42] [WARNING] Setting sync_ext to True by default for syncing changes
[16:47:42] [INFO] Loading jurigged
[16:47:42] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\Main.py
[16:47:42] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\play.py
[16:47:42] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\embed.py
[16:47:42] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\askbard.py
[16:47:42] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\noi_chu.py
[16:47:42] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\db_refesh.py
[16:47:42] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\help.py
[16:47:42] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\channel_listen.py
[16:47:42] [DEBUG] Caching ingested rate limit data for: 0a6d458104b103c7be293cc9b284d9ee
[16:47:42] [DEBUG] GET /gateway Received 200 :: [99/100 calls remaining]
[16:47:42] [INFO] Shard ID 0 | Starting Shard
[16:47:42] [INFO] Shard ID 0 | Shard is attempting to connect to gateway...
[16:47:42] [DEBUG] Shard ID 0 | Identification payload sent to gateway, requesting intents: 3407613
[16:47:43] [INFO] Shard ID 0 | Gateway connection established
[16:47:43] [DEBUG] Shard ID 0 | Session ID: 5a1305b33a4546825b94fa4b7d242739 Trace: ['["gateway-prd-us-east1-b-g22j",{"micros":175865,"calls":["id_created",{"micros":503,"calls":[]},"session_lookup_time",{"micros":295,"calls":[]},"session_lookup_finished",{"micros":15,"calls":[]},"discord-sessions-prd-2-259",{"micros":173702,"calls":["start_session",{"micros":95033,"calls":["discord-api-6c869fb97d-ddnvm",{"micros":81049,"calls":["get_user",{"micros":11157},"get_guilds",{"micros":11672},"send_scheduled_deletion_message",{"micros":10},"guild_join_requests",{"micros":2},"authorized_ip_coro",{"micros":11}]}]},"starting_guild_connect",{"micros":58,"calls":[]},"presence_started",{"micros":10292,"calls":[]},"guilds_started",{"micros":198,"calls":[]},"guilds_connect",{"micros":2,"calls":[]},"presence_connect",{"micros":68090,"calls":[]},"connect_finished",{"micros":68097,"calls":[]},"build_ready",{"micros":21,"calls":[]},"clean_ready",{"micros":1,"calls":[]},"optimize_ready",{"micros":0,"calls":[]},"split_ready",{"micros":1,"calls":[]}]}]}]']
[16:47:43] [DEBUG] Dispatching Event: websocket_ready
[16:47:43] [DEBUG] Dispatching Event: guild_join
[16:47:43] [DEBUG] Dispatching Event: guild_join
[16:47:43] [DEBUG] Dispatching Event: guild_join
[16:47:43] [DEBUG] Dispatching Event: guild_join
[16:47:43] [DEBUG] Dispatching Event: guild_join
[16:47:43] [DEBUG] Caching ingested rate limit data for: dd84d9818020c27975c0d5ab4a794d44
[16:47:43] [DEBUG] GET /applications/{application_id}/commands Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[16:47:43] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [0/1000 calls remaining]
[16:47:43] [DEBUG] Waiting for bucket dd84d9818020c27975c0d5ab4a794d44 to unlock.
[16:47:43] [DEBUG] Sending heartbeat every 41.25 seconds
[16:47:43] [DEBUG] Sending data to websocket: {"op": 1, "d": 6}
[16:47:43] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[16:47:43] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[16:47:44] [DEBUG] Caching ingested rate limit data for: dd84d9818020c27975c0d5ab4a794d44
[16:47:44] [DEBUG] GET /applications/{application_id}/commands Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[16:47:44] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [0/1000 calls remaining]
[16:47:44] [DEBUG] 0 is already up-to-date with 14 commands.
[16:47:44] [DEBUG] Sync of 1 scopes took 1.0463065000076313 seconds
[16:47:44] [DEBUG] Dispatching Event: startup
[16:47:44] [DEBUG] Sending data to websocket: {"op": 3, "d": {"since": 1703756864343, "activities": [{"name": "/help for helpful", "type": 0}], "status": "idle", "afk": false}}
[16:47:45] [DEBUG] 1174695146214928500::play should be called
[16:47:46] [DEBUG] Dispatching Event: message_create
[16:47:46] [DEBUG] Caching ingested rate limit data for: 432f6223265d156004ac81366c5a948f
[16:47:46] [DEBUG] POST /interactions/{interaction_id}/aW50ZXJhY3Rpb246MTE4OTg2NzI4MDY4Nzk2ODI1Njp2Y3lBaHhocE9jVzFCdlRHRUxCZmFXOXFjNXhQVFRXZVU3bVdnYmZNVWF2MUtxQzA3cGgwaXZiRXoyd0IwbnZxd0g4dFZxYjFEbnpuQkFnMWowQVhzNWRYMXF5Nnp1M0FIRFhMMkk0RUVsdjVQbmJWSUZhYnI2M1VZd05wRHNTNw/callback Received 204 :: [4/5 calls remaining]
[16:47:46] [DEBUG] Sending data to websocket: {"op": 4, "d": {"guild_id": 1067024293709496340, "channel_id": 1189851990356918383, "self_mute": false, "self_deaf": false}}
[16:47:46] [DEBUG] Waiting for voice connection data...
[16:47:46] [DEBUG] Dispatching Event: voice_user_join
[16:47:46] [DEBUG] Attempting to initialise voice gateway...
[16:47:47] [DEBUG] Voice Connection has identified itself to Voice Gateway
[16:47:47] [DEBUG] Discord send VC Ready! Establishing a socket connection...
[16:47:47] [DEBUG] IP Discovery in progress...
[16:47:47] [DEBUG] Voice Initial Response Received: b'\x00\x02\x00F\x00\x05c\x99116.100.183.123\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xc3\xd9'
[16:47:47] [DEBUG] IP Discovered: 116.100.183.123 #50137
[16:47:47] [DEBUG] Sending data to websocket: {"op": 1, "d": {"protocol": "udp", "data": {"address": "116.100.183.123", "port": 50137, "mode": "xsalsa20_poly1305_lite"}}}
[16:47:47] [DEBUG] Starting UDP Keep Alive
[16:47:47] [DEBUG] Unhandled OPCODE: 18 = data = {'user_id': '610841389164396565', 'flags': 2}
[16:47:47] [DEBUG] Unhandled OPCODE: 20 = data = {'user_id': '610841389164396565', 'platform': 0}
[16:47:47] [INFO] Voice connection established; using xsalsa20_poly1305_lite
[16:47:48] [DEBUG] Sending heartbeat every 13.75 seconds
[16:47:48] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.9715008823048453}
[16:47:48] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:47:48] [DEBUG] ❤ Heartbeat acknowledged after 0.09588 seconds
[16:47:48] [DEBUG] Dispatching Event: command_error
[16:47:48] [DEBUG] Dispatching Event: command_completion
[16:47:48] [DEBUG] Dispatching Event: error
[16:47:48] [INFO] Command Called: play with event.ctx.args = ['https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'] | event.ctx.kwargs = {'song': 'https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'}
[16:47:48] [ERROR] Ignoring exception in cmd `/play`:
Traceback (most recent call last):
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1900, in __dispatch_interaction
    response = await callback
               ^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1771, in _run_slash_command
    return await command(ctx, **ctx.kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\command.py", line 132, in __call__
    await self.call_callback(self.callback, context)
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\application_commands.py", line 833, in call_callback
    return await self.call_with_binding(callback, ctx, *new_args, **new_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\callback.py", line 43, in call_with_binding
    return await callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\music\play.py", line 132, in play
    await music_queues.playlist_process(url_list)
  File "D:\Project\Python\qpneruy-Git\modules\Music_queue.py", line 39, in playlist_process
    audio = await youtube_dl.get_audio(song)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\modules\YT_Downloader.py", line 54, in get_audio
    data = await asyncio.to_thread(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\asyncio\threads.py", line 25, in to_thread
    return await loop.run_in_executor(None, func_call)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\concurrent\futures\thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\modules\YT_Downloader.py", line 55, in <lambda>
    lambda: cfg_video.extract_info(url, download=False)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\yt_dlp\YoutubeDL.py", line 1556, in extract_info
    if not ie.suitable(url):
           ^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\yt_dlp\extractor\lazy_extractors.py", line 152, in suitable
    qs = parse_qs(url)
         ^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\yt_dlp\utils\_utils.py", line 2484, in parse_qs
    return urllib.parse.parse_qs(urllib.parse.urlparse(url).query, **kwargs)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\urllib\parse.py", line 394, in urlparse
    url, scheme, _coerce_result = _coerce_args(url, scheme)
                                  ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\urllib\parse.py", line 133, in _coerce_args
    return _decode_args(args) + (_encode_result,)
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\urllib\parse.py", line 117, in _decode_args
    return tuple(x.decode(encoding, errors) if x else '' for x in args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\urllib\parse.py", line 117, in <genexpr>
    return tuple(x.decode(encoding, errors) if x else '' for x in args)
                 ^^^^^^^^
AttributeError: 'coroutine' object has no attribute 'decode'

[16:48:01] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.020974641468537336}
[16:48:01] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:48:01] [DEBUG] ❤ Heartbeat acknowledged after 0.09162 seconds
[16:48:15] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.02405936015772636}
[16:48:15] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:48:15] [DEBUG] ❤ Heartbeat acknowledged after 0.09019 seconds
[16:48:24] [DEBUG] Sending data to websocket: {"op": 1, "d": 10}
[16:48:24] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[16:48:25] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[16:48:26] [DEBUG] Dispatching Event: disconnect
[16:48:26] [DEBUG] Stopping the bot.
[16:48:26] [INFO] Shard ID 0 | Stopping Shard
